index.html

```diff
@@ -42,6 +42,208 @@ <h2 class="connect">LET'S CONNECT</h2>
+          <span class="tag">Back End Dev</span>
+          <span class="bullet"></span>
+          <span class="tag"> 2015</span>
+          
+        </div>
+       
+        <div class="descreption" >
+          <p> A daily selection of privately
+              personalized reads; no accounts or 
+              sign-up required.</p>
+          </div>
+
+            <div class="languages">
+              <ul>
+                <li  class="language-tag">html</li>
+                <li  class="language-tag">css</li>
+                <li  class="language-tag">Javascript</li>
+              </ul>
+            </div>
+
+            <button class="see-project">See Project</button>
+          </div>
+       
+      </div>
+      </div>
+      <section class="last-section">
+        <div>
+          <h1 class="intro">About <br />Myself</h1>
+        </div>
+        <div>
+          <p class="about-me">
+            I’m a software developer! I can help you <br />
+            build a product , feature or website Look <br />
+            through some of my work and <br />
+            experience! If you like what you see and <br />
+            have a project you need coded, don’t <br />
+            hestiate to contact me.
+          </p>
+        </div>
+        <div class="contact-me">
+          <h2 class="connect">LET'S CONNECT</h2>
+        </div>
+        <div class="my-social-link">
+          <ul class="buttons">
+          <li><a href=""><img src="/assets/Github.svg"  class="contact-btn" alt="Github" /></li></a>
+          <li><a href=""><img src="/assets/Twitter.svg" class="contact-btn"  alt="Twitter" /></li></a>
+          <li><a href=""><img src="/assets/medium.svg"  class="contact-btn" alt="Medium" /></li></a>
+          <li><a href=""><img src="/assets/Vector.svg"  class="contact-btn" alt="" /></li></a>
+        </ul>
+        </div>
+        <div class="get-my-resume">
+          <button class="see-project resume">Get My resume</button>
+        </div>
+        
+      </section>
+    </section>
+    <div class="grid-container">
+      <div class="my-skills">
+        
+      </div>
+    </div>
+    <div class="vector">
+      <h4>Languages</h4>
+      <img src="/assets/Union.svg" alt="">
+      </div>
+      <div class ="mastring-languages">
+        <ul>
+          <li class="ms-language"><img class="lg-icons" src="/assets/Ellipse 1.png"/>Javascript</i</li>
+          <li class="ms-language"><img class="lg-icons" src="/assets/HTML.png"/>HTML</i</li>
+          <li class="ms-language"><img class="lg-icons" src="/assets/Css.png"/>CSS</i</li>
+        </ul>
+      </div>
+      <div class="vector">
+        <h4>Frameworks</h4>
+        <img src="/assets/Disabled.svg" alt="">
+        </div>
+        <div><hr class="black-line"></div>
+        <div class="vector">
+          <h4>Skills</h4>
+          <img src="/assets/Disabled.svg" alt="">
+          </div>
+          <div><hr class="black-line"></div>
+          
```

Comment:
- [ ] Kindly make sure that the technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li, that is for sematic reason :+1: